the-collab-lab/tcl-63-smart-shopping-list
# Focus add item

## Description

The goal was to bring focus back to the input field after every submission.

## Acceptance Criteria
- [x] Pressing the Enter key or the Add Item button on the AddItem page should always bring the focus back to the input field.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria

To pull the branch locally, run git pull origin focus-add-item
Next, run git checkout focus-add-item
Install dependencies npm install
Launch the app with npm start

- Enter several items, one at a time.
- Alternate between pressing the Enter key or the Add Item button.
- The focus should always be back to the input field.
